EventsExpress.Db/Migrations/20210203142212_AddSalt.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace EventsExpress.Db.Migrations
+{
+    public partial class AddSalt : Migration
```

Comment:
This migration is invalid - there is no `Salt` column in the database. It will fail when deploying to dev. Consider generating new migration from the dev branch instead from you database state.

Reply:
So, why is invalid?
1. There is no Salt column in the database now
2. We need it, so add field and create a migration
3. And the Up method will create it

That looks correct to me